tests/unit/sync/transformTaxonomyGroupModel.test.ts

```diff
@@ -0,0 +1,76 @@
+import { describe, expect, it } from "@jest/globals";
+import { TaxonomyContracts } from "@kontent-ai/management-sdk";
+
+import { transformTaxonomyGroupsModel } from "../../../src/modules/sync/modelTransfomers/taxonomyGroups";
+
+const input: ReadonlyArray<TaxonomyContracts.ITaxonomyContract> = [
+  {
+    id: "taxonomyGroup!Id",
+    last_modified: "groupModified1",
+    name: "taxonomyGroup1",
+    codename: "taxonomy_group1",
+    external_id: "extId_taxonomy_group1",
+    terms: [
+      {
+        id: "nestedTermId1",
+        name: "nestedTerm",
+        codename: "nested_term",
+        last_modified: "nestedTermModofied",
+        terms: [],
+      },
+    ],
+  },
+  {
+    id: "taxonomyGroup2Id",
+    last_modified: "groupModified2",
+    name: "taxonomyGroup2",
+    codename: "taxonomy_group2",
+    external_id: "extId_taxonomy_group2",
+    terms: [
+      {
+        id: "nestedTermId2",
+        name: "nestedTerm2",
+        codename: "nested_term2",
+        last_modified: "nestedTermModofied2",
+        external_id: "extId_nested_term2",
+        terms: [],
+      },
+    ],
+  },
+] as const;
+
+const expectedOutput = [
```

Comment:
I am not sure about the expected input/output being outside the test case. What if I'll want to add another test case later? Would I be forced to add something like `expectedOutputForTheSecondCase`? I would prefer the data to be part of the `it` callback even though they are pretty long to avoid the problem. The data is not that long after all.

Reply:
I did it because of indentation - wanted to save some space :D for me, there is only one test. When somebody wants to add new test, he might then change it so the test would contain its own expectedOutput :)

Reply:
Ok, we can leave it be for now.